### PR TITLE
fix corner case of expander when empty array occurs in structured lg

### PR DESCRIPTION
**Pull request by feich-ms**
fix https://github.com/microsoft/botbuilder-dotnet/issues/3973

### CODE REVIEW

File: tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/Examples/Expand.lg [hunk 1]
```diff
@@ -100,4 +100,16 @@ They are ${ShowAlarm(alarms[1])}
 [MyStruct
     Speak = ${GetAge()}
     Text = zoo
+]
+
+# PointOfInterestSuggestedActionNameText(Data)
```
Comment (xieofxie) [inline tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/Examples/Expand.lg:105]: Maybe you could change the name to make it a more formal test name

Comment (feich-ms) [reply tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/Examples/Expand.lg:105]: The name seems ok I think.
